pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.heigit.ohsome</groupId>
     <artifactId>ohsome-parent</artifactId>
-    <version>2.5</version>
+    <version>2.6</version>
```

Comment:
let's wait for the fix in https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/parent/-/blob/master/pom.xml#L120